### ISSUE TITLE
Logits and log_probs set to 0 when using cross_entropy

### ISSUE THREAD

**Issue by datduonguva** [tensorflow/models#8398]
In MaskGan, seq2seq_vd, if the training strategy is `cross_entropy`, all the logits, ,log_probs output of the generator's decoder are set to 0. Why is this the case?
https://github.com/tensorflow/models/blob/0672bdcf4c06768155e30632dd0763c1f9a3d002/research/maskgan/models/seq2seq_vd.py#L354

@a-dai
